app-profile-jee-vanilla/pom.xml

```diff
@@ -77,6 +77,12 @@
                     <skip>false</skip>
                 </configuration>
             </plugin>
+            
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-war-plugin</artifactId>
+                <version>${version.war.maven.plugin}</version>
+            </plugin>
```

Comment:
BTW - is this needed just for this `app-profile-jee-vanilla` quickstart? I can see failures related to the plugin when building other applications too, e.g.: `app-jee-html5`